pkgs/by-name/ne/netmaker-pro/package.nix

```diff
@@ -0,0 +1 @@
+{ netmaker }: netmaker.override { isPro = true; }
```

Comment:
I don't believe it is a good idea to "abuse" by-name hierarchy this way. 
It is cleaner to put this as an alias on top-level/all-packages.nix instead.

Reply:
I was not possible for me due to those errors:
```
pkgs.netmaker-pro: This is a new top-level package of the form `callPackage ./pkgs/by-name/ne/netmaker/package.nix { }`. Please define it in pkgs/by-name/ne/netmaker-pro/package.nix instead. See `pkgs/by-name/README.md` for more details. Since the second `callPackage` argument is not `{ }`, the manual `callPackage` (e.g. in `pkgs/top-level/all-packages.nix`) is still needed.
pkgs.nmctl: This is a new top-level package of the form `callPackage ./pkgs/by-name/ne/netmaker/package.nix { }`. Please define it in pkgs/by-name/nm/nmctl/package.nix instead. See `pkgs/by-name/README.md` for more details. Since the second `callPackage` argument is not `{ }`, the manual `callPackage` (e.g. in `pkgs/top-level/all-packages.nix`) is still needed.
```

Reply:
actually nevermind, I think I used it wrongly